server/etcdserver/server.go

```diff
@@ -856,6 +857,7 @@ func (s *EtcdServer) run() {
 				s.sync(s.Cfg.ReqTimeout())
 			}
 		case <-s.stop:
+			lg.Warn("server stopped")
```

Comment:
I agree. 

This is just to confirm the theory during debugging. `Member remove/list` requests was sent to the removed member that was stopped.

client side retry intercepter logging is misleading because the target is always the first endpoint when dialing with 3 endpoints. https://github.com/etcd-io/etcd/blob/5b7b499f5007f0cdbe91f781ff91691c2dbc527c/client/v3/retry_interceptor.go#L55-L60